utils/call-gen/src/upload_program.rs

```diff
@@ -29,7 +29,7 @@ type UploadProgramArgsInner = (Vec<u8>, Vec<u8>, Vec<u8>, u64, u128);
 /// Upload program args
 ///
 /// Main type used to generate arguments for the `pallet_gear::Pallet::<T>::upload_program` call.
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, PartialEq, Eq)]
```

Comment:
mb add for future other default traits like Default and Ords?

Reply:
Will add Ords, but not `Default`, as it looks wrong. How can we decide which GearCall is Default? I mean, the impl of the trait won't be meaningful, it will just help bypass type-level boundaries.

Reply:
[a3b1603](https://github.com/gear-tech/gear/pull/3117/commits/a3b160371c299e258eb8eb2b98e3eda5ea3eac1d)